model/src/main/scala/cromwell/pipeline/model/validator/Wrapped.scala

```diff
@@ -4,40 +4,34 @@ import cats.Show
 import cats.data.NonEmptyList
 import cats.data.Validated.{ Invalid, Valid }
 import cats.data.{ NonEmptyChain, Validated }
-import play.api.libs.json.{ Format, JsError, JsResult, JsSuccess, JsValue, Json }
+import play.api.libs.json.{ Format, JsResult, JsSuccess, JsValue, Json }
 
 import scala.util.control.NoStackTrace
 import scala.language.implicitConversions
+import org.slf4j.LoggerFactory
 
-trait Wrapped[T] extends Any {
-  def unwrap: T
+trait Wrapped extends Any {
+  def unwrap: String
```

Comment:
I haven't watched the whole request yet, but I accidentally noticed it. I don't know what you are trying to do, but you are doing it wrong.
`Wrapped` should be able to wrap values of any given type and provide some useful logic.

Reply:
fixed.